docs/user/sbt.rst

```diff
@@ -85,7 +85,8 @@ Garbage collectors
 
    Garbage collector that allocates things without ever freeing them. Useful
    for short-running command-line applications or applications where garbage
-   collections pauses are not acceptable.
+   collections pauses are not acceptable. The gc maps 1TB of memory using
+   mmap. It performs no bound checks and fails once it runs out of memory.
```

Comment:
1TB might seem like a lot now, but I'm pretty sure there are people who already have machines with more memory than that. What's the performance win for not doing the bounds check?